components/chassis.py

```diff
@@ -57,6 +60,9 @@ def __init__(
             ctre.FeedbackDevice.IntegratedSensor, 0, 10
         )
 
+        self.encoder = encoder
+        self.encoder.setPosition(encoder_offset)
```

Comment:
Does this have an effect in absolute mode? I thought it only lets you set the number of wraparounds.

Reply:
fixed, also added comments for what to change to for cancoders